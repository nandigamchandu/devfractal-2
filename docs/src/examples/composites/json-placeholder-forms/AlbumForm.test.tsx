import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { AlbumForm } from './AlbumForm'

test('Album Form', () => {
  const component: ReactTestRenderer = renderer.create(<AlbumForm />)
  expect(component.toJSON()).toMatchSnapshot()
})
