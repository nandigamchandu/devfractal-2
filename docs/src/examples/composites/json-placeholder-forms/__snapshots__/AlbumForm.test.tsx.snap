// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Album Form 1`] = `
<section
  className="section"
>
  <h1
    className="title"
  >
    Album Form
  </h1>
  <form
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label
        className="label"
      >
        Title
      </label>
      <div
        className="control"
      >
        <input
          className="input"
          name="title"
          onBlur={[Function]}
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="field is-grouped is-grouped-centered"
    >
      <div
        className="control"
      >
        <button
          className="button is-info"
          disabled={false}
          type="submit"
        >
          Submit
        </button>
      </div>
      <div
        className="control"
      >
        <button
          className="button is-danger"
          disabled={true}
          onClick={[Function]}
          type="reset"
        >
          Reset
        </button>
      </div>
    </div>
    <pre>
      {
  "title": ""
}
    </pre>
  </form>
</section>
`;
